domain/src/main/kotlin/sample/jpa/report/model/entity/Report.kt

```diff
@@ -0,0 +1,39 @@
+package sample.jpa.report.model.entity
+
+import sample.jpa.CreatedAtEntity
+import sample.jpa.post.model.entity.Post
+import sample.jpa.report.model.enum.ReportState
+import sample.jpa.report.model.enum.ReportType
+import sample.jpa.users.model.entity.User
+import javax.persistence.*
+
+@Entity
+class Report (
+        @Id
+        @GeneratedValue(strategy = GenerationType.IDENTITY)
+        val id: Long? = 0,
+
+        @Column(name = "title", length = 100, nullable = false)
+        var title: String,
+
+        @Lob
+        @Column(name = "content", nullable = true)
+        var content: String,
```

Comment:
생각해보니 이거 `nullable = true`이면 타입도 nullable해야 하지 않나요??
```suggestion
        @Lob
        @Column(name = "content", nullable = true)
        var content: String?,
```

Reply:
어 그렇네 맞아용 수정하겠습니다 !